zenpy/__init__.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, subdomain=None,
                  session=None,
                  timeout=None,
                  ratelimit=None,
-                 ratelimit_budget=None):
+                 ratelimit_budget=None,
+                 request_interval=None):
```

Comment:
Maybe call this `ratelimit_request_interval` to make it clear that it is related to the ratelimit  functionality.